magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -14,6 +15,8 @@ def slurm_lines(queue, job_name, array=None, mem=None, out_name=None):
         Name of the queue
     job_name : str
         Job name
+    nice_parameter : int
+        Job priority
```

Comment:
you implemented both comments that were partially clashing. since  --nice line is meant to be added only when you specify this parameter, you can still allow None (and put it as a default in the function) as it was before